src/main/webapp/app/core/user/settings/user-settings-container/user-settings-container.component.spec.ts

```
@@ -11,8 +11,10 @@
 import { MockAccountService } from 'test/helpers/mocks/service/mock-account.service';
 import { MockActivatedRoute } from 'test/helpers/mocks/activated-route/mock-activated-route';
 import { UserSettingsContainerComponent } from 'app/core/user/settings/user-settings-container/user-settings-container.component';
+import { ProfileService } from 'app/core/layouts/profiles/shared/profile.service';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ProfileService'. Consider to delete it or rename it.

'ProfileService' is defined here:


(lint/suspicious/noRedeclare)